documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/modules-standard.adoc

```diff
@@ -17,7 +17,7 @@
 include::module-alpn.adoc[]
 include::module-bytebufferpool.adoc[]
 include::module-console-capture.adoc[]
-include::module-deploy.adoc[]
+include::module-eeN-deploy.adoc[]
```

Comment:
@sbordet: As far as I can tell, the functionality of the old `deploy` module has been mostly rolled into the `eeN-deploy` modules. For this reason, I renamed the documentation file.

However, there still seems to be a `deploy` module (i.e., without the `eeN` prefix) [hanging around](https://github.com/gregpoulos/jetty.project/blob/jetty-12.0.x-document-modules/jetty-core/jetty-deploy/src/main/config/modules/deploy.mod). It's not clear to me what purpose it serves; it seems to be a transitive dependency, and not something that the end-user will need to look at very often. For this reason, I don't think it makes sense to document it for the time being.

Reply:
The `deploy` module is an "abstract" base module that the `eeN-deploy` modules depend on.
Agreed that it's "internal" and does not need docs.